### PR TITLE
feat(ui5-panel): noAnimation property introduced

### PR DESCRIPTION
**Pull request by niyap**
New property called noAnimation is added to the ui5-panel component. When set to true, the ui5-panel is expanded/collapsed without animation.

FIXES: #3505

### CODE REVIEW

File: packages/main/src/Panel.js [hunk 1]
```diff
@@ -90,6 +90,17 @@ const metadata = {
 			type: Boolean,
 		},
 
+		/**
+		 * Indicates whether the transition between the expanded and the collapsed state of the component is animated. By default the animation is enabled.
+		 *
+		 * @type {boolean}
+		 * @defaultvalue false
```
Comment (ilhan007) [inline packages/main/src/Panel.js:97]: add since tag 1.0.0-rc.16

**Review by ilhan007 — APPROVED** [review]

**Comment by ilhan007** [comment]
add since tag to the new property, otherwise looks fine +1

File: packages/main/src/Panel.js (hunk 1 above)
Comment (niyap) [reply packages/main/src/Panel.js:97]: Done

**Review by MapTo0 — DISMISSED** [review]

**Review by ilhan007 — APPROVED** [review]

**Comment by ilhan007** [comment]
eslint

/home/travis/build/SAP/ui5-webcomponents/packages/main/src/Panel.js
  102:17  error  Missing trailing comma  comma-dangle

**Review by MapTo0 — APPROVED** [review]

**Pull request merged**